### ISSUE TITLE
RepoDB overall refactoring of Tracing capabilities

### ISSUE THREAD

**Issue by mikependon** [cajuncoding/HotChocolate.RepoDb#10]
First and foremost, you have contributed your thoughts to this story already, and, we are thanking you for that. Allow us to use this as the means of notification as you are one of the major user.

Anyway, please be informed that we are about to start the development of this [User Story](https://github.com/mikependon/RepoDB/issues/941).

We would like to capture any comments from your side (if there are), prior to the development activity.

The changes will be part of the next "minor" release (> RepoDB v1.12.10), even it is breaking changes.

**Comment by cajuncoding** [comment]
Still need to dig into these changes (eventually), for now I'm sticking with the current lower version... but will leave this open until I can come back around to it.

**Comment by mikependon** [comment]
Yeah, there are as well other breaking changes when you move the version up. Better to stabilize the solution first and branchout with the newest versions if you are planning.
